lib/dradis/plugins/nexpose/formats/full.rb

```diff
@@ -141,15 +142,17 @@ def process_full(doc)
 
         # 3.2 associate with the nodes via Evidence.
         #   TODO: there is room for improvement here by providing proper Evidence content
-        xml_vulnerability.xpath('./hosts/host').collect(&:text).each do |host_name|
+        xml_vulnerability.xpath('./hosts/host').map(&:text).each do |host_name|
```

Comment:
What's the reason for this change?

Reply:
https://github.com/rubocop/ruby-style-guide#map-find-select-reduce-include-size although not a hard requirement